pkg/checks/checks.go

```diff
@@ -69,19 +69,19 @@ type Check interface {
 
 type Result struct {
 	// data contains performance metrics about the check run
-	Data any `json:"data"`
+	Data any `json:"data" yaml:"data"`
 	// Timestamp is the UTC time the check was run
-	Timestamp time.Time `json:"timestamp"`
+	Timestamp time.Time `json:"timestamp" yaml:"timestamp"`
 	// Err should be nil if the check ran successfully indicating the check is "healthy"
 	// if the check failed, this should be an error message that will be logged and returned to an API user
-	Err string `json:"error"`
+	Err string `json:"error" yaml:"error"`
 }
```

Comment:
Why did you apply the yaml tags to the structs even if they have nothing to do with the config?